tests/testthat/test-data_tabulate.R

```diff
@@ -405,3 +405,59 @@ test_that("data_tabulate, correct 0% for proportions", {
   expect_identical(format(out[[1]])[[4]], c("0 (0%)", "0 (0%)", "0 (0%)", "0 (0%)", "", "0"))
   expect_snapshot(print(out[[1]]))
 })
+
+
+# coercing to data frame -------------------------
+
+test_that("data_tabulate, as.data.frame", {
```

Comment:
Can you break this big `test_that()` into several smaller ones based where you added comments?